runtime/navigator.js

```diff
@@ -99,9 +99,8 @@ function createEventListeners(updatePage) {
 }
 
 function handleClick(event) {
-  const el = event.target.closest('a')
+  const el = event.target.closest('a') || event.composedPath().find(el => el.tagName === 'A')
```

Comment:
Should it be `el.tagName === 'a'`?

Reply:
TagName returns the DOM tag in uppercase all the time.

Reply:
Ah, no worries